### PR TITLE
Modal adapts to content changing height

### PR DESCRIPTION
**Pull request by kennyt**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

File: src/Modal/ModalContents.js [hunk 1]
```diff
@@ -63,8 +64,22 @@ export default class ModalContents extends Util.mixin(BindMixin) {
   }
 
   checkHeight() {
+    let prevContentHeight = this.heightInfo.innerContentHeight;
```
Comment (mlunoe) [inline src/Modal/ModalContents.js]: Could we get some tests in for this function?
I wanna make sure that we are doing it right with these extra calculations

**Comment by mlunoe** [comment]
I cannot come up with a more elegant solution right now, and I am guessing this is an intermediate solution, until we can spend some more time on it?

I really hoped we wouldn't make this component more complex than it is

**Comment by kennyt** [comment]
yeah, I think we should do a height calculation overhaul later on. but surprisingly, the modal has been reliable (other than the content height change, which this should fix)

File: src/Modal/ModalContents.js (hunk 1 above)
Comment (kennyt) [inline src/Modal/ModalContents.js]: alright, made some tests

**Pull request merged**